checkstyle_config.xml

```diff
@@ -120,17 +120,19 @@
                value="GenericWhitespace ''{0}'' is not preceded with whitespace."/>
     </module>
     <module name="Indentation">
-      <property name="basicOffset" value="4"/>
+      <property name="basicOffset" value="2"/>
```

Comment:
We use indentation of 2 instead of 4 on our current project. If you wish, you may leave 4 here and on next 2 lines as well.